trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class TrimmerDecorator < Nameable
+  def initialize(nameable = Nameable.new)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- This method should return the trimmed name if the name is longer than 10 chars, else it should return the name. Please add a condition to return the name if it is not more than 10 chars 👍 👍

Reply:
Thank you, I have worked on it.